modules/RTC/JitsiLocalTrack.js

```diff
@@ -104,7 +104,7 @@ export default class JitsiLocalTrack extends JitsiTrack {
         /**
          * The value of bytes sent received from the statistics module.
          */
-        this._bytesSent = null;
+        this._bytesSentPerPC = new Map();
```

Comment:
if we're just going to add up all the bytes sent for each peer connection, why make this a map? it could still just be a number which we add to every time, no?

Reply:
Hmm. When we receive the new value it should represent the bytes since the beginning of the conference, isn't it? If we just add them every time I believe we won't get the total bytes sent but something else.

Reply:
ok, i didn't know what the value we were receiving was, but here we really only care that the number is > 0, right?  so this could even just be a boolean `haveReceivedDataFromMicrophone` which would be set to true any time `_setBytesSent` was called with a value `> 0`

Reply:
@bbaldino It is boolean now!